### DISCUSSION TITLE
Set username in updateAccount bound to a specific number?

### DISCUSSION THREAD

**Discussion by bbernhard** [AsamK/signal-cli#1486] [General]
Hi,

I am currently playing around with the `updateAccount` command and in particular with the recently added `-u` flag which allows to set the username. Initially I thought, that I could set different usernames for different accounts. So basically something like:

`signal-cli updateAccount -a <mynumber> -u test`

But it seems that `-a` is not allowed here. Is that something that will be allowed at some point or is there a technical restriction that prevents that from working?

Thanks a lot!

**Comment by AsamK** [comment]
The `-a` is a global parameter, so needs to be in front of the command: `signal-cli -a <mynumber> updateAccount -u test`

**Reply by bbernhard** [reply]
Oh man, this is embarrassing. I've done that like hundred times and now did such a stupid mistake. :man_facepalming: 

Thanks a lot - and sorry for the noise!
